src/messages/en.json

```diff
@@ -19,5 +19,82 @@
         "sponsor_pt1": "THANK YOU TO OUR SPONSORS",
         "sponsor_pt2": "FOR THEIR GENEROUS SUPPORT",
         "include": "Designed & Developed with 🤍 by #include @ Davis"
+    },
+    "Services": {
+        "heading": "Our Services",
+        "services1": {
```

Comment:
I would just change the name of the string here to something like `topServices` so it doesn't accidentally get confused with `services1`